rollup.config.js

```diff
@@ -1,41 +1,46 @@
 /**
- * Shared config used by individual packages' Rollup configs. Items here come in three flavors:
- *   - stand-alone: used by `@sentry/browser`, `@sentry/tracing`, and `@sentry/vue` (bundles which are a full SDK in
- *     and of themselves)
- *   - add-on: used by `@sentry/integrations` and `@sentry/wasm` (bundles which need to be combined with a stand-alone
- *     SDK bundle)
- *   - shared: used by both types of bundles
- *
+ * Code for generating config used by individual packages' Rollup configs
  */
 
+import assert from 'assert';
+
 import deepMerge from 'deepmerge';
 import license from 'rollup-plugin-license';
 import resolve from '@rollup/plugin-node-resolve';
 import replace from '@rollup/plugin-replace';
 import { terser } from 'rollup-plugin-terser';
 import typescript from 'rollup-plugin-typescript2';
 
-export const paths = {
-  '@sentry/browser': ['../browser/src'],
-  '@sentry/core': ['../core/src'],
-  '@sentry/hub': ['../hub/src'],
-  '@sentry/minimal': ['../minimal/src'],
-  '@sentry/types': ['../types/src'],
-  '@sentry/utils': ['../utils/src'],
-};
+/**
+ * Helper functions to compensate for the fact that JS can't handle negative array indices very well
+ *
+ * TODO `insertAt` is only exported so the integrations config can inject the `commonjs` plugin, for localforage (used
+ * in the offline plugin). Once that's fixed to no longer be necessary, this can stop being exported.
+ */
+function getLastElement(array) {
+  return array[array.length - 1];
+}
+export function insertAt(arr, index, insertee) {
```

Comment:
This doesn't prettier?

```suggestion
}

export function insertAt(arr, index, insertee) {
```

Reply:
Do you mean, does prettier get mad that there's not a blank line there? No - it would have failed linting. My thinking in not putting a blank there is I wanted the comment to apply to both, and it felt like they were both small enough I could get away with it. But you're not wrong, it's a little weird looking. Compromise option:

```js
/**
 * Helper functions to compensate for the fact that JS can't handle negative array indices very well
 *
 * TODO `insertAt` is only exported so the integrations config can inject the `commonjs` plugin, for localforage (used
 * in the offline plugin). Once that's fixed to no longer be necessary, this can stop being exported.
 */
const getLastElement = array => {
  return array[array.length - 1];
};
export const insertAt = (arr, index, insertee) => {
  const newArr = [...arr];
  // Add 1 to the array length so that the inserted element ends up in the right spot with respect to the length of the
  // new array (which will be one element longer), rather than that of the current array
  const destinationIndex = index >= 0 ? index : arr.length + 1 + index;
  newArr.splice(destinationIndex, 0, insertee);
  return newArr;
};
```

Functionally* the same, but having two consts in a row is better. I'll switch it to that.

\* no pun intended, I swear